### ISSUE TITLE
Organization of scattering section

### ISSUE THREAD

**Issue by janden** [mathieulagrange/paperStructureScene16#3]
Describing the first-order scattering coefficients from some generic time-frequency decomposition x_1 before going back and defining what x_1 is makes this section hard to follow. In addition, we can't say much about the resulting "first-order scattering coefficients", since we need the wavelet structure of x_1 to derive any stability estimates. I propose that we reorganize this section to talk a little about invariance and stability in general, then define x_1, S_1 x, etc. in the normal order.

**Comment by mathieulagrange** [comment]
+1 as the presentation of invariance and stability was an issue raised by the reviewers

**Comment by janden** [comment]
Ok will do.
